dev/NavigationView/NavigationView.cpp

```diff
@@ -1779,10 +1779,90 @@ void NavigationView::OnNavigationViewItemKeyDown(const winrt::IInspectable& send
     }
 }
 
+void NavigationView::ArrowKeyNavigationPolyfill(const winrt::NavigationViewItem& nvi, const winrt::KeyRoutedEventArgs& args)
+{
+    switch (args.Key())
```

Comment:
>Key() [](start = 17, length = 5)

This code will execute for gamepad up and down, is that intended?

Reply:
I'm having trouble verifying whether gamepad up and down behaves as expected in RS2 without these changes, however this is the intended behavior for gamepad up and down as well.